### ISSUE TITLE
Windows wheel issue

### ISSUE THREAD

**Issue by illume** [viblo/pymunk#156]
Hello,

I noticed this issue with latest windows release of pymunk 5.4.1 (python 3.7 with windows 32bit)
https://ci.appveyor.com/project/pygame/stuntcat/builds/21400786

Here is the relevant error message:
```
error: cannot load library 'C:\Python37\lib\site-packages\pymunk\chipmunk.dll': error 0x7e
```

For now I'm working around it by declaring the requirements differently on different platforms.
```
pymunk>=5.4.1; sys_platform != 'win32'
pymunk==5.4.0; sys_platform == 'win32'
```

**Comment by viblo** [comment]
Thanks for the report. However, I cant reproduce it myself. Trying it now (win10, 32bit python 3.7.2) and then pip install pymunk it works fine and loads the dll just as it should. Does the same thing happen if you run outside of appveyor?

**Comment by viblo** [comment]
Or wait a minute.. is it something to do with failing while inside bdist_msi maybe?

**Comment by illume** [comment]
Ah. It's failing with this command:
```
%PYTHON%\python.exe setup.py bdist_msi
```

**Comment by illume** [comment]
I'll have to wait for my windows laptop to charge up before testing.

**Comment by illume** [comment]
Hi again,

someone else also encountered the error, and told me they were using this python version:
```
3.6.3 (v3.6.3:2c5fed8, Oct  3 2017, 17:26:49) [MSC v.1900 32 bit (Intel)]
```

**Comment by viblo** [comment]
I think its fixed in new release 5.4.2 Would be great if you could try and see if it works better now. 

I managed to do some debugging myself, on my work laptop I got the same error as you. It seems to be because 5.4.1 was built with mingw-w64 from appveyor, while the previous ones was built locally by tdm-gcc. mingw-w64 seems to link to libwinpthread-1.dll, while I guess tdm did it statically (I didnt confirm exactly what tdm-gcc does).  
Anyway, since this seemed like a serious problem I made a quick fix release, pymunk 5.4.2 with the threaded solver disabled on windows. With this fix it no longer needs to link against the pthread library.

The threaded solver makes it so that you can run the space.step() function on two threads instead of one, if configured. However, for most real world use cases I dont think the benefit is very huge (if any), so should be ok in most cases.

**Comment by illume** [comment]
It works!  Thank you very much :) https://ci.appveyor.com/project/pygame/stuntcat
